### PR TITLE
7.0-mgmtsystem_nonconformity_name

### PR DESCRIPTION
**Pull request by foutoucour**
```
* Added name field in tree view and form view
* Hide description field in the list view
* Replaced description by name in form view header
```

### CODE REVIEW

File: mgmtsystem_nonconformity/mgmtsystem_nonconformity.xml [hunk 1]
```diff
@@ -86,6 +87,7 @@
                     <group>
                         <group>
                         	<field name="ref" attrs="{'readonly':[('state','not in',['draft'])]}"/>
+                            <field name="name" attrs="{'readonly':[('state','not in',['draft'])]}"/>
```
Comment (dreispt) [inline mgmtsystem_nonconformity/mgmtsystem_nonconformity.xml:90]: The title can be always visible, so IMO we can remove the attrs.

File: mgmtsystem_nonconformity/mgmtsystem_nonconformity.py [hunk 2]
```diff
@@ -172,6 +172,7 @@ def _state_name(self, cr, uid, ids, name, args, context=None):
         # 1. Description
         'id': fields.integer('ID', readonly=True),
         'ref': fields.char('Reference', size=64, required=True, readonly=True),
+        'name': fields.char('Name', required=True),
```
Comment (dreispt) [inline mgmtsystem_nonconformity/mgmtsystem_nonconformity.py:175]: I suggest the `name` label do be "Subject", but other opinions are welcome.

File: mgmtsystem_nonconformity/mgmtsystem_nonconformity.xml (hunk 1 above)
Comment (foutoucour) [inline mgmtsystem_nonconformity/mgmtsystem_nonconformity.xml:90]: Hey @dreispt , do you mean always writable? The attrs acts only on readonly here.
I am confused sorry :(

File: mgmtsystem_nonconformity/mgmtsystem_nonconformity.py (hunk 2 above)
Comment (foutoucour) [inline mgmtsystem_nonconformity/mgmtsystem_nonconformity.py:175]: @dreispt do you think it stop the merge?

Comment (dreispt) [inline mgmtsystem_nonconformity/mgmtsystem_nonconformity.py:175]: Not at all; not even sure others agree on that label.

File: mgmtsystem_nonconformity/mgmtsystem_nonconformity.xml (hunk 1 above)
Comment (dreispt) [inline mgmtsystem_nonconformity/mgmtsystem_nonconformity.xml:90]: My bad, sorry. `attrs` for `readonly` needs to be there.

**Comment by dreispt** [comment]
:+1: LGTM

**Comment by bwrsandman** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**